xline/src/server/xline_server.rs

```diff
@@ -73,6 +79,7 @@ impl XlineServer {
         key_pair: Option<(EncodingKey, DecodingKey)>,
         server_timeout: ServerTimeout,
         client_timeout: ClientTimeout,
+        mem_storage: S,
```

Comment:
I don't think we should expose too many details about the implementation in a variable name. Please modify `mem_storage` as `storage`.

Reply:
my mistake, fixed